ci/taos/plugins-staging/pr-nnstreamer-apptest.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+##
+# Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+##
+# @file pr-nnstreamer-apptest.sh
+# @brief Check nnstreamer sample app working well.
+# @see      https://github.com/nnsuite/TAOS-CI
+# @author   Sewon Oh <sewon.oh@samsung.com>
+
+
+# @brief [MODULE] TAOS/pr-nnstreamer-apptest-trigger-queue
+function pr-nnstreamer-apptest-trigger-queue(){
```

Comment:
How about we try to rename -postfix from now on as following:
1. wait-queue
2. run-queue 
3. ...

Reply:
In case of qeue name, let's do the modification with another PR.

Reply:
https://github.com/nnsuite/TAOS-CI/pull/203 is merged for improving the queue structure.